pkg/R/DataFrame.R

```diff
@@ -768,6 +798,20 @@ setMethod("select", signature(x = "DataFrame", col = "Column"),
             dataFrame(sdf)
           })
 
+setMethod("select",
+          signature(x = "DataFrame", col = "list"),
```

Comment:
FWIW in R you can use `do.call` to call a varargs function with a list of arguments. For example
```
arglist <- list(x=runif(10), trim=0.1, na.rm=TRUE)
do.call(mean, arglist)
```